packages/bpk-component-calendar/src/BpkColoredCalendarDate.js

```diff
@@ -32,25 +34,30 @@ export const CellType = {
   positive: 'positive',
 };
 
-// TODO: Rename cellStyle to cellType
-const BpkColoredCalendarDate = props => {
-  const { cellType, ...rest } = props;
+type Props = {
```

Comment:
We need to export this here and in the index file as well

Reply:
Why do we need to do this? Its only used internally to the component so doesn't need to be external?

Reply:
Props are a external part of the component's API so we need to expose it, people might need/want to use it for various reasons, but if we remove this then no need :)